pegasus/sites.v3/code.org/public/index.haml

```diff
@@ -29,7 +29,7 @@ social:
   = view :homepage_hero, languageFilter: languageFilter
 
   #petitionexpand{style: "display: none"}
-    %script{:src=>"/js/jquery.placeholder.js"}
+    %script{:defer =>"defer",:src=>"/js/jquery.placeholder.js"}
```

Comment:
This is relied upon by any calls to `.placeholder()`. I see two calls in the [`:petition_expand` view](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/views/petition_expand.haml#L86-L87) in `Petition()`, which is apparently only called during `$(document).ready`. As far as I can tell this _should_ occur after any `defer`-loaded javascript, so this should be okay.